tests/acceptance/lib/files/set_variable_values_ini.cf

```diff
@@ -34,6 +34,9 @@ bundle agent test
       "description" -> { "CFE-3221" }
         string => "Test expected behavior of set_variable_values_ini";
 
+      "description" -> { "CFE-3244" }
```

Comment:
Instead of re-setting the description here, please move the promisee/stakeholder ( "CFE-3244" ) to the previous one.

```
"description" -> { "CFE-3221", "CFE-3244" }
```

Reply:
Also, should have similar change for manage_variable_values_ini.cf